security/pkg/nodeagent/sds/sdsservice.go

```diff
@@ -44,6 +45,9 @@ const (
 	// credentialTokenHeaderKey is the header key in gPRC header which is used to
 	// pass credential token from envoy to SDS.
 	credentialTokenHeaderKey = "authorization"
+
+	// uriScheme is the URI scheme for Istio identities.
+	uriScheme = "spiffe"
```

Comment:
You can reference it here. https://github.com/istio/istio/blob/master/security/pkg/pki/util/san.go#L31

Reply:
updated.